### PR TITLE
creating templates list

### PR DESCRIPTION
**Pull request by aidanatwork**
## Purpose / why

To enable templateers to quickly see what new templates still need to be created, and their priority. Fixes [issue 199](https://github.com/thegooddocsproject/templates/issues/199).

## What changes were made?

Created new file with a bulleted list.

## Verification

Make sure you agree with the idea of adding this file, and provide feedback on the specific contents.

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Are issues linked correctly?
* [ ] Is this PR labeled correctly?
* [ ] If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?
* [ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [ ] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?
* [ ] On merging, did you add any applicable notes to a [draft release](https://github.com/thegooddocsproject/templates/releases) and link to this PR?

### CODE REVIEW

**Comment by aidanatwork** [comment]
Heads up that per discussion in the last Base Template Working Group standup, I will also be putting the base template back at the top-level as a P0.

File: templates-list.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	* P3 Project Governance - RFC processes?
+	* P3 License (Recommendations on attribution)
+* *Guide*
+	* P3 Style guide (for code)
+	* P3 Style guide (for documentation)
+* P3 Release notes
+* P4 Logging
+* P4 Pull request template
+* P4 Bug reporting template
+* P4 Comments (structured comments you can grok and grep)
```
Comment (barbaricyawps) [inline templates-list.md]: **Question (non-blocking):** I'm not sure what this template would be? It looks like it's based on the Conventional Comments guide?

File: templates-list.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	* P3 Issue template and merge request template
+	* P3 Project Governance - RFC processes?
+	* P3 License (Recommendations on attribution)
+* *Guide*
+	* P3 Style guide (for code)
+	* P3 Style guide (for documentation)
+* P3 Release notes
+* P4 Logging
+* P4 Pull request template
+* P4 Bug reporting template
```
Comment (barbaricyawps) [inline templates-list.md]: **Question (non-blocking):** It looks like the bug reporting template is a duplicate of the P3 Issue template in the Community section. Also, I recommend making the issue template and the merge request template should be two separate templates. They're very different from each other functionally.

**Review by barbaricyawps — APPROVED** [review]
Looks good to me. I have some minor comments, questions, and suggestions but nothing is blocking. Therefore, approved.

File: templates-list.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# List of Templates to Create
+
+The list below includes all the templates we as templateers plan to create. Feel free to provide feedback on whether any templates need to be removed or added to the list. The "P{number}" before each item indicates the priority of completing this template. We want to focus mostly or now on completing P1 templates, but if you have a special interest in one of the other templates feel free to work on that too. Feedback on the priorities is also welcome. Items without "P{number}" are just categories, not templates to be created.
```
Comment (camerons) [inline templates-list.md:3]: Nitpick (non-blocking):
```suggestion
The list below includes all the templates we as templateers plan to create. Feel free to provide feedback on whether any templates need to be removed or added to the list. The "P{number}" before each item indicates the priority of completing this template. We want to focus mostly on completing P1 templates, but if you have a special interest in one of the other templates feel free to work on that too. Feedback on the priorities is also welcome. Items without "P{number}" are just categories, not templates to be created.
```

**Review by camerons — APPROVED** [review]
Looks good. One nitpick suggested rewording.
I've approved. Hopefully that will give Aiden sufficient access to apply the fix and then merge the pull request.

**Pull request merged**